src/associates/src/saa-scss/saa-associates.scss

```diff
@@ -0,0 +1,90 @@
+//@import '../../../scss/config/index.scss';
+
+header,
+.newMembersToggle {
+  margin-left: 0;
+}
+
+.newMembersToggle {
+  input {
+    height: 18px;
+    width: 18px;
+    border: 2px solid #919294;
```

Comment:
this is the blue color, could  be changed to cardinalred,

I think that integrating Decanter for modular typography and color variable names can be a separate task that we come back to. I think it can be something we clean up later in August after higher-priority items. I'm in favor of merging this branch after a few tweaks and making a small 'clean up react widget' task since the main font is applied in some places.


@rebeccahongsf please apply Proxima Nova font to the main text of the widget - input text in the input and dropdown, New Member checkbox label, A-B letter tabs at the top, names and content in the table, etc.